Assets/MixedRealityToolkit/Services/BaseCoreSystem.cs

```diff
@@ -1,39 +1,42 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-
 namespace Microsoft.MixedReality.Toolkit
 {
     public abstract class BaseCoreSystem : BaseEventSystem
     {
         /// <summary>
         /// Constructor.
         /// </summary>
-        /// <param name="registrar">The <see cref="IMixedRealityServiceRegistrar"/> instance that loaded the service.</param>
         /// <param name="profile">The configuration profile for the service.</param>
-        [System.Obsolete("This constructor is obsolete (registrar parameter is no longer required) and will be removed in a future version of the Microsoft Mixed Reality Toolkit.")]
         protected BaseCoreSystem(
-            IMixedRealityServiceRegistrar registrar,
-            BaseMixedRealityProfile profile = null) : this(profile)
+            BaseMixedRealityProfile profile = null) : base()
         {
-            Registrar = registrar;
+            ConfigurationProfile = profile;
+            Priority = 5; // Core systems have a higher default priority than other services
         }
 
+        #region Obsolete
```

Comment:
> #region Obsolete [](start = 7, length = 17)

Just to check, all that was done here was a reorder (i.e moving the Obsolete constructor second, adding it to an Obsolete region) right?

The diffs are actually fairly hard to read despite being what seems like just a reorder

Reply:
yes, sorry I tried to call that out in the description

Reply:
gah, that's my bad for missing that then.

---
In reply to: [370892079](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7145#discussion_r370892079) [](ancestors = 370892079)